lib/translation_io/config.rb

```diff
@@ -6,9 +6,14 @@ class Config
     attr_accessor :verbose
     attr_accessor :test
 
+    attr_accessor :multi_domain, :domain_names, :domain_api_keys
+    attr_accessor :domain_source_locales, :domain_target_locales, :domain_folders
+    attr_accessor :text_domain
+
     attr_accessor :ignored_key_prefixes
     attr_accessor :ignored_source_paths
     attr_accessor :ignored_source_files
+    attr_accessor :forced_source_paths
```

Comment:
Code to force certain path prefixes was cleaner with an opt-in path list than with the ignored, opt-out which existed